ruleset/decoders/0040-auditd_decoders.xml

```diff
@@ -1,362 +1,360 @@
+  <parent>auditd</parent>
+  <regex offset="after_regex">old-auid=(\S+) auid=(\S+) old-ses=(\S+) ses=(\S+) res=(\S+)</regex>
+  <order>audit.old-auid,audit.auid,audit.old-ses,audit.session,audit.res</order>
 </decoder>
 
 <!--
-Generic
-type=TEST_GENERIC msg=audit(1234567890.123:1234): addr=10.10.10.10 ses=20 exe="ls" comm="ls" ppid=432 pid=3333 auid=0 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0
+  Generic
+  type=TEST_GENERIC msg=audit(1234567890.123:1234): addr=10.10.10.10 ses=20 exe="ls" comm="ls" ppid=432 pid=3333 auid=0 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0
 -->
 <decoder name="auditd-generic">
-    <parent>auditd</parent>
-    <regex offset="after_parent">^(\S+) msg=audit\(\d\d\d\d\d\d\d\d\d\d.\d\d\d:(\d+)\): </regex>
-    <order>audit.type,audit.id</order>
+  <parent>auditd</parent>
+  <regex offset="after_parent">^(\S+) msg=audit\(\d\d\d\d\d\d\d\d\d\d.\d\d\d:(\d+)\): </regex>
+  <order>audit.type,audit.id</order>
 </decoder>
 
 <decoder name="auditd-generic">
-    <parent>auditd</parent>
-    <regex>pid=(\S+)</regex>
-    <order>audit.pid</order>
+  <parent>auditd</parent>
+  <regex>pid=(\S+)</regex>
+  <order>audit.pid</order>
 </decoder>
 
 <decoder name="auditd-generic">
-    <parent>auditd</parent>
-    <regex>auid=(\S+)</regex>
-    <order>audit.auid</order>
+  <parent>auditd</parent>
+  <regex>auid=(\S+)</regex>
+  <order>audit.auid</order>
 </decoder>
 
 <decoder name="auditd-generic">
-    <parent>auditd</parent>
-    <regex> uid=(\S+)</regex>
-    <order>audit.uid</order>
+  <parent>auditd</parent>
+  <regex> uid=(\S+)</regex>
+  <order>audit.uid</order>
 </decoder>
 
 <decoder name="auditd-generic">
-    <parent>auditd</parent>
-    <regex>gid=(\S+)</regex>
-    <order>audit.gid</order>
+  <parent>auditd</parent>
+  <regex>gid=(\S+)</regex>
+  <order>audit.gid</order>
 </decoder>
 
 <decoder name="auditd-generic">
-    <parent>auditd</parent>
-    <regex>euid=(\S+)</regex>
-    <order>audit.euid</order>
+  <parent>auditd</parent>
+  <regex>euid=(\S+)</regex>
+  <order>audit.euid</order>
 </decoder>
 
 <decoder name="auditd-generic">
-    <parent>auditd</parent>
-    <regex>ses=(\S+)</regex>
-    <order>audit.session</order>
+  <parent>auditd</parent>
+  <regex>ses=(\S+)</regex>
+  <order>audit.session</order>
 </decoder>
 
 <decoder name="auditd-generic">
-    <parent>auditd</parent>
-    <regex>comm=\p(\S+)\p</regex>
-    <order>audit.command</order>
+  <parent>auditd</parent>
+  <regex>comm=\p(\S+)\p</regex>
+  <order>audit.command</order>
 </decoder>
 
 <decoder name="auditd-generic">
-    <parent>auditd</parent>
-    <regex>exe=\p(\S+)\p</regex>
-    <order>audit.exe</order>
+  <parent>auditd</parent>
+  <regex>exe=\p(\S+)\p</regex>
+  <order>audit.exe</order>
 </decoder>
 
 <decoder name="auditd-generic">
-    <parent>auditd</parent>
-    <regex>addr=(\S+)</regex>
-    <order>audit.srcip</order>
+  <parent>auditd</parent>
+  <regex>addr=(\S+)</regex>
+  <order>audit.srcip</order>
```

Comment:
use srcip as field name instead of audit.srcip, our template doesn't has audit.srcip as valid field.